trollsched/schedule.py

```diff
@@ -572,12 +642,12 @@ def parse_datetime(strtime):
     return datetime.strptime(strtime, "%Y%m%d%H%M%S")
 
 
-def save_passes(allpasses, poly, output_dir):
+def save_passes(allpasses, poly, output_dir, plot_parameters, plot_title):
```

Comment:
Maybe make the two last parameters optional not to break backwards compatibility.